### PR TITLE
Fix the Elasticsearch Operator install based on slack

### PR DESCRIPTION
**Pull request by mburke5678**
https://coreos.slack.com/archives/CB3HXM2QK/p1559834747040200

### CODE REVIEW

File: modules/efk-logging-deploy-subscription.adoc [hunk 1]
```diff
@@ -55,11 +60,49 @@ metadata:
 ----
 <1> You must specify the `openshift-operators-redhat` namespace.
 
-.. Create the Namespace object:
+.. Run the following command to create the project:
```
Comment (sichvoge) [inline modules/efk-logging-deploy-subscription.adoc]: "project" should be "namespace"

File: modules/efk-logging-deploy-subscription.adoc [hunk 2]
```diff
@@ -35,13 +35,18 @@ You *must* install the Elasticsearch Operator using the CLI following the direct
...
-. Install the Elasticsearch Operator using the CLI to ensure the necessary values are set:
+. Create Namespaces for the Elasticsearch Operator and Cluster Logging Operator.
++
+[NOTE]
+====
+You can create the Namespaces in the web console using the *Administration* -> *Namespaces* page.
+====
 
-.. Create a Namespace for the Elasticsearch operator (for example, `eo-project.yaml`):
+.. Create a Namespace for the Elasticsearch Operator (for example, `eo-project.yaml`):
```
Comment (sichvoge) [inline modules/efk-logging-deploy-subscription.adoc]: would probably change the example to `eo-namespace.yaml` just to comply with us always using "namespace" and not "project".

File: modules/efk-logging-deploy-subscription.adoc [hunk 3]
```diff
@@ -55,11 +60,49 @@ metadata:
...
+$ oc create -f <file-name>.yaml
+----
++
+For example:
 +
 ----
 $ oc create -f eo-project.yaml
 ----
+  
+.. Create a Namespace for the Cluster Logging Operator (for example, `clo-project.yaml`):
```
Comment (sichvoge) [inline modules/efk-logging-deploy-subscription.adoc]: Same as in my previous comment, change example to `clo-namespace.yaml`.

File: modules/efk-logging-deploy-subscription.adoc [hunk 4]
```diff
@@ -55,11 +60,49 @@ metadata:
...
+    openshift.io/node-selector: "" <1>
+  labels:
+    openshift.io/cluster-logging: "true"
+    openshift.io/cluster-monitoring: "true"
+----
+<1> Optionally specify an empty node selector in order for the logging pods to spread
+evenly across your cluster. The logging pods will be spread evenly throughout the cluster.
+If you want the logging pods to run on specific nodes, you can specify a node selector value here.
+
+.. Run the following command to create the project:
```
Comment (sichvoge) [inline modules/efk-logging-deploy-subscription.adoc]: "project" should be "namespace"

File: modules/efk-logging-deploy-subscription.adoc [hunk 5]
```diff
@@ -55,11 +60,49 @@ metadata:
...
++
+----
+$ oc create -f <file-name>.yaml
+----
++
+For example:
++
+----
+$ oc create -f clo-project.yaml
```
Comment (sichvoge) [inline modules/efk-logging-deploy-subscription.adoc]: `clo-namespace.yaml`

File: modules/efk-logging-deploy-subscription.adoc [hunk 6]
```diff
@@ -55,11 +60,49 @@ metadata:
...
+.. Run the following command to create the project:
++
+----
+$ oc create -f <file-name>.yaml
+----
++
+For example:
 +
 ----
 $ oc create -f eo-project.yaml
```
Comment (sichvoge) [inline modules/efk-logging-deploy-subscription.adoc]: `eo-namespace.yaml`

**Review by sichvoge — CHANGES_REQUESTED** [review]

File: modules/efk-logging-deploy-subscription.adoc [hunk 7]
```diff
@@ -35,13 +35,18 @@ You *must* install the Elasticsearch Operator using the CLI following the direct
...
 
-
 .Procedure
 
-. Install the Elasticsearch Operator using the CLI to ensure the necessary values are set:
+. Create Namespaces for the Elasticsearch Operator and Cluster Logging Operator.
++
+[NOTE]
+====
+You can create the Namespaces in the web console using the *Administration* -> *Namespaces* page.
```
Comment (bparees) [inline modules/efk-logging-deploy-subscription.adoc]: "but you must ensure the cluster-logging and cluster-monitoring labels listed in the sample yaml content below are applied to the namespaces you create"

Comment (bparees) [reply modules/efk-logging-deploy-subscription.adoc]: s/can/can also/

File: modules/efk-logging-deploy-subscription.adoc (hunk 4 above)
Comment (jcantrill) [reply modules/efk-logging-deploy-subscription.adoc]: Clould we also not do this with a series of oc commands like:
```
oc create ns openshift-logging
oc label ns openshift-logging openshift.io/cluster-logging="true" openshift.io/cluster-monitoring="true"
```

Comment (bparees) [reply modules/efk-logging-deploy-subscription.adoc]: sounds plausible

Comment (mburke5678) [reply modules/efk-logging-deploy-subscription.adoc]: @jcantrill I believe that we need `openshift.io/node-selector: ""` in the CLO namespace, which is not created by `oc create`. Can we inject that parameter via the CLI?

**Comment by mburke5678** [comment]
@anpingli PTAL

**Review by anpingli — COMMENTED** [review]
The yam content  tagged using the digit 1),  but the explatination is a.  Shall we use 1) instead of a. ?
apiVersion: v1
kind: Namespace
metadata:
  name: openshift-operators-redhat (1)
a.  You must specify the openshift-operators-redhat namespace.

**Comment by mburke5678** [comment]
@anpingli I think that GitHub renders the `<1>` as `a.` in their viewer. I hadn't noticed this before. 
The `<1>` appears correctly in this preview:
http://file.rdu.redhat.com/mburke/logging-install-fix-eo/logging/efk-logging-deploying.html#efk-logging-deploy-subscription_efk-logging-deploying

**Pull request merged**